### PR TITLE
moved to DFD3: lambdas are not special symbols, and datastore labels …

### PR DESCRIPTION
**Pull request by izar**
…appear outside of shape

### CODE REVIEW

**Comment by izar** [comment]
dfd3

File: pytm/pytm.py [hunk 1]
```diff
@@ -1354,19 +1351,34 @@ def __init__(self, name, **kwargs):
...
+        levels = kwargs.get("levels", None)
+        if levels and not levels & self.levels:
+            return ""
+
+        return self._dfd_template().format(
+            uniq_name=self._uniq_name(),
+            label=self._label(),
+            color=self._color(),
+            shape=self._shape(),
+            image=os.path.join(os.path.dirname(__file__), "images", "datastore.png"),
```
Comment (nineinchnick) [inline pytm/pytm.py:1376]: This image would look better if the top and bottom would not touch, that is if it would be a bit taller.

**Review by nineinchnick — APPROVED** [review]
Yay! Less HTML tables in labels!

File: pytm/pytm.py (hunk 1 above)
Comment (izar) [reply pytm/pytm.py:1376]: sample

Comment (izar) [reply pytm/pytm.py:1376]: oh just noticed the flow names are broken now. Dammit graphviz.

Comment (izar) [reply pytm/pytm.py:1376]: sample

Comment (colesmj) [reply pytm/pytm.py:1376]: @izar Where is the code that renders the arrows here? Is the shaded vs unshaded arrow heads the implementation of the initiator mark?

Comment (nineinchnick) [reply pytm/pytm.py:1376]: It's a combination of the `arrowtail` and `dir` options. `arrowtail` is always set to `onormal` (open) but is not drawn by default, when `dir` is `forward`. `dir` is `both` if a request and response are merged together into a single flow.

**Comment by izar** [comment]
Talked with @colesmj and made sure he ok's the merge.

**Pull request merged**